node/risk-admin/server/routes/shp2json.js

```diff
@@ -20,24 +18,18 @@ module.exports = {
       const zipfile = tmpfile + '.zip'
       await rename(tmpfile, zipfile)
 
-      const { data: geojson } = await ogr2ogr(zipfile)
+      let data
+      try {
+        ({ data } = await ogr2ogr(zipfile))
```

Comment:
Are the outer brackets required here?

Reply:
Yes, usually you would do
`const { data } = await ogr2ogr(zipfile)`
and that just copies the data field from the object returned by the call into the new const.

However, as this time I'm not using a const, but an already declared variable (because of the block scoping rules when wrapping the call in a try {} block) just doing
`{ data } = await ogr2ogr(zipfile)`
Isn't valid syntax, so you have to enclose the statement in brackets.

Took me a while to work out how to do it, but eventually found the language spec for destructuring, which has an example.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment